packages/payload/src/versions/saveVersion.ts

```diff
@@ -34,6 +34,7 @@ export const saveVersion = async ({
   const versionData = deepCopyObject(doc)
   if (draft) versionData._status = 'draft'
   if (versionData._id) delete versionData._id
+  if (versionData.id) delete versionData.id
```

Comment:
I'm not confident that this change won't affect the subsequent process, so I'd appreciate it if you could review that point specifically.